pricejar/base/templates/base/home.html

```diff
@@ -2,8 +2,370 @@
 
 {% block content %}
 
```

Comment:
Just after the {% block content %} tag add the head tag content of the home page. The head content has the styling and bootstrap links required. Make sure you djangofy your file links